src/modules/account/components/BackupRecoveryPhraseFlow/BackupRecoveryPhraseFlow.test.js

```diff
@@ -0,0 +1,78 @@
+import { fireEvent, screen, waitFor } from '@testing-library/react';
+import { renderWithRouter } from 'src/utils/testHelpers';
+import * as reactRedux from 'react-redux';
```

Comment:
@gadjacobs Please ensure that package imports are placed before aliased and relative imports